run-pytype.py

```diff
@@ -34,7 +32,8 @@ def run_pytype(command: List[str], branch_url: str, errorlog: TextIO, results):
                 printfrom = line.index("]") + 2
             info("PROGRESS: " + line[1:].split("]")[0] + ": " + line[printfrom:])
             continue
-        elif line.startswith("ninja: "):
+
+        if line.startswith("ninja: "):
```

Comment:
Why was this changed?

Reply:
Is is changed to fix this pylint report (Using "elif" after a "continue" is not necessary):

> Unnecessary "elif" after "continue", remove the leading "el" from "elif" (no-else-continue)